source/Calamari.Shared/Calamari.Shared.csproj

```diff
@@ -84,7 +84,7 @@
     <PackageReference Include="Microsoft.Web.Deployment" Version="3.6.0" />
     <PackageReference Include="NuGet.CommandLine" Version="2.8.6" />
     <PackageReference Include="NuGet.Core" Version="2.14.0" />
-    <PackageReference Include="AlphaFS" Version="2.1.3-octopus0006" />
+    <PackageReference Include="AlphaFS" Version="2.2.6" />
```

Comment:
I'm a big fan of updating external dependencies where we can, given proper consideration to the risks, so usually I'd be all thumbs up here.
The old version number `2.1.3-octopus0006` made me think it might be a custom Octopus branch though, and indeed it does look like it: https://github.com/OctopusDeploy/AlphaFS

Reply:
Reverting this to the original version. We'll phase out AlphaFS once we're off the .NET Framework.